packages/testcafe-models/cardView/index.ts

```diff
@@ -102,6 +107,15 @@ export default class CardView extends GridCore {
     return this.element.find(`.${CLASS.cardViewContent}`).hasClass(`${CLASS.selectCheckBoxesHidden}`);
   }
 
+  isReady(): Promise<boolean> {
+    const { getInstance } = this;
+
+    return ClientFunction(
+      () => (getInstance() as any).isReady(),
```

Comment:
@dmlvr minor: could we avoid type assertion operator here?

Reply:
@siarhei-arzamasau, I fixed it. Now it works without type assertion operator

```
  isReady(): Promise<unknown> {
    const { getInstance } = this;

    return ClientFunction(
      () => {
        const instance = getInstance();
        if (typeof instance === 'object' && instance !== null && 'isReady' in instance && typeof instance.isReady === 'function') {
          return instance.isReady();
        }
      },
      { dependencies: { getInstance } },
    )();
  }
```